.eslintrc.js

```diff
@@ -60,4 +60,10 @@ module.exports = {
       "error", { "beforeColon": false, "afterColon": true },
     ],
   },
+  "settings": {
+    "react": {
+      // This is what Spotify uses
+      "version": "16.0",
```

Comment:
<img width="219" alt="image" src="https://user-images.githubusercontent.com/1565883/175779284-b9093188-e3c2-46fa-96ef-191ca942897e.png">

Reply:
Interesting, since the version number is what I got from vendor~xpui. Will change.

Reply:
Oh weird. And this is from 1.1.86 as well.